platform/security/sts/security-sts-crlinterceptor/src/test/java/org/codice/ddf/security/sts/crl/CRLInterceptorTest.java

```diff
@@ -39,152 +39,58 @@
  */
 public class CRLInterceptorTest {
 
-    private static final String REVOKED_CERT = "/certs/ia-bad.crt";
-
-    private static final String VALID_CERT = "/certs/ia.crt";
-
-    private static final String CRL_LOCATION = CRLInterceptorTest.class
-            .getResource("/certs/root.crl").getFile();
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(CRLInterceptorTest.class);
-
-    // START Error cases (calling interceptor should throw AccessDeniedException)
-
-    /**
-     * Verifies that a certificate designated as revoked in the CRL is properly
-     * caught in the interceptor.
-     *
-     */
-    @Test(expected = AccessDeniedException.class)
-    public void testDenyCertificate() {
-        Message message = null;
-        try {
-            message = createMockMessageWithCert(REVOKED_CERT);
-        } catch (CertificateException ce) {
-            LOGGER.error(
-                    "Could not create certificate objects from files located in test, failing test.",
-                    ce);
-            fail();
-        }
-
-        callNewInterceptor(CRL_LOCATION, true, message);
-
-    }
-
     /**
-     * Tests that the interceptor will deny access if it is enabled and there is
-     * no CRL file set.
+     * Tests that the interceptor will NOT throw an error if CRL is disabled
      */
-    @Test(expected = AccessDeniedException.class)
-    public void testDenyNoCRL() {
-        Message message = null;
-        try {
-            message = createMockMessageWithCert(VALID_CERT);
-        } catch (CertificateException ce) {
-            LOGGER.error(
-                    "Could not create certificate objects from files located in test, failing test.",
-                    ce);
-            fail();
-        }
-
-        // create and call interceptor manually (without setting the CRL)
-        CRLInterceptor interceptor = new CRLInterceptor();
-        interceptor.setIsEnabled(true);
-        interceptor.handleMessage(message);
-    }
+    @Test
+    public void testRevokedCertWithCrlDisabledPasses() throws CertificateException {
+        Message message = createMockMessageWithCert(getRevokedCert());
+        CRLInterceptor crlInterceptor = new CRLInterceptor();
+        crlInterceptor.crlChecker.setCrlLocation(null);
```

Comment:
Why not inject a mock instead, have `CrlChecker.passesCrlCheck()` return `true` or `false` depending on the test and verify that it's been called?

Assuming that we create a separate test class for the `CrlChecker`, we could simplify this test class a whole lot because all we would need to test are the `true` and `false` scenarios. Right now, we're pretty much testing `CrlChecker` twice, once here and once in the `PKHHandlerTest`.